src/user/ZulipButton.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  TouchableHighlight,
+  ActivityIndicator,
+} from 'react-native';
+
+const styles = StyleSheet.create({
+  buttonFrame: {
+    flex: 1,
+    alignItems: 'center',
+    padding: 12,
+    height: 44,
+    backgroundColor: '#22693F',
+    borderRadius: 5,
+    width: 260,
+  },
+  buttonText: {
+    color: '#FFFFFF',
+  },
+});
+
+export default class ZulipButton extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
State is not needed. I think it is left from refactoring the previous buttons into this one.
Also adding Flow validation for props, which will be a good idea for all other components too.